xbmc/utils/CharsetConverter.cpp

```diff
@@ -21,8 +21,11 @@
 #include "CharsetConverter.h"
 
 #include <algorithm>
-
+#ifndef TARGET_FREEBSD
 #include <iconv.h>
+#elif TARGET_FREEBSD
+#include "/usr/include/iconv.h"
+#endif
```

Comment:
Is this still needed on top of latest master?

Reply:
FreeBSD's own original iconv.h header (at '/usr/local/include') is not correct. It would compile without errors but fail while linking.

This is GNU stub header. Just "portability" include. 
With TARGET_FREEBSD KODI builder seeks it's defined includes from '/usr/local/include/'.

I'll keep in mind what you ask.  I have some of it already done. I'll try to have more acceptable outcome with next push.